### PR TITLE
Update the hover effect and background colors of links

### LINKED ISSUE #186: Update the hover effect and background colors of links
Before :
image

After :
image
image
image

### PR DESCRIPTION
**Pull request by Kunal0007**
Fixes #186 

## Proposed Changes

  - Updated the background colors of links on hover
  - Updated the scaling effect on links

## Screenshots
Before :
image

After :
image
image
image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/195"></a>

### CODE REVIEW

File: src/index.css
Comment (rohan-kulkarni-25) [inline src/index.css:20]: Does the 0.05 difference matter ? IF not let's keep 1.02 as its rounded of.

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]
- Please provide after and before snaps of changes 
- GIF if possible

File: src/index.css
Comment (Kunal0007) [reply src/index.css:20]: It does not matter much but it looks better than before.

File: src/index.css
Comment (BOLT04) [inline src/index.css:36]: are these `!important` necessary? In my opinion we should try to avoid using it unless it's absolutely necessary. In this case it's a simple case of **specificity** (you can check [these docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception) to know more).
How about we do this:
```suggestion
.p-button.p-button-outlined:not(a):not(.p-disabled).twitter:hover {
  background-color: #1DA1F2;
  color: #fff !important;
}

.p-button.p-button-outlined:not(a):not(.p-disabled).youtube:hover {
  background-color: #FF0000;
  color: #fff !important;
}

.p-button.p-button-outlined:not(a):not(.p-disabled).github:hover {
  background-color: #211F1F;
  color: #fff !important;
}
```

I didn't remove the `color` property `!important` because we are using inline styles... which I think should be refactored too 😅, but changes should be made incrementally and not all in one go 👍  
@Kunal0007 what do you think 🙂?

File: src/index.css
Comment (BOLT04) [reply src/index.css:20]: if it's not a big difference, I'd go with the one that looks best 👍 .

**Review by BOLT04 — COMMENTED** [review]
Looks good, great work @Kunal0007 👍. Just left a few comments, I'd love to get your thoughts on them 😃

File: src/index.css
Comment (Kunal0007) [reply src/index.css:36]: Thank you for your suggestions, great work 👍. Yes, changes should be made incrementally. Someone other can make an issue or a PR for these changes it will increase contributions 🙂.

**Review by BOLT04 — APPROVED** [review]
LGTM 👍

File: src/index.css
Comment (eddiejaoude) [reply src/index.css:20]: I agree 👍

File: src/index.css
Comment (eddiejaoude) [reply src/index.css]: I prefer to not have `!important` if possible too, these sorts of improvements won't happen later

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the [**merge conflicts**](https://github.com/EddieHubCommunity/LinkFree/pull/195/conflicts).

File: src/index.css
Comment (Panquesito7) [inline src/index.css:44]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍
I'll let @eddiejaoude review before merging, though.

**Comment by Kunal0007** [comment]
@Panquesito7 Sure, can you tell me how to solve this https://github.com/EddieHubCommunity/LinkFree/pull/195#discussion_r716182549

**Comment by Panquesito7** [comment]
> @Panquesito7 Sure, can you tell me how to solve this [#195 (comment)](https://github.com/EddieHubCommunity/LinkFree/pull/195#discussion_r716182549)

https://github.com/EddieHubCommunity/LinkFree/pull/195#discussion_r711273042

**Comment by Kunal0007** [comment]
@Panquesito7 Thank you, I made some changes to remove !important in index.css.

File: src/index.css
Comment (Kunal0007) [reply src/index.css]: @eddiejaoude made some changes

**Comment by eddiejaoude** [comment]
Great collaboration everyone 👍

**Comment by Kunal0007** [comment]
Thank you everyone for suggesting changes and @eddiejaoude for merging it ☺️

**Pull request merged**
